lib/ReactCrop.jsx

```diff
@@ -509,37 +509,54 @@ var ReactCrop = React.createClass({
 		);
 	},
 
+	arrayDividedBy100(arr, delimeter) {
+		delimeter = delimeter || ' ';
+		return arr.map(number => number / 100).join(delimeter);
+	},
+
 	arrayToPercent(arr, delimeter) {
 		delimeter = delimeter || ' ';
 		return arr.map(number => number + '%').join(delimeter);
 	},
 
-	getImageClipStyle() {
+	getPolygonValues(forSvg) {
 		let crop = this.state.crop;
-
-		let right = 100 - (crop.x + crop.width);
-		let bottom = 100 - (crop.y + crop.height);
-
-		// Safari doesn't like it if values add up to exactly
-		// 100 (it doesn't draw the clip). I have submitted a bug.
-		if (crop.x + right === 100) {
-			right -= 0.00001;
+		let pTopLeft = [ crop.x, crop.y ];
+		let pTopRight = [ crop.x + crop.width, crop.y ];
+		let pBottomLeft = [ crop.x, crop.y + crop.height ];
+		let pBottomRight = [ crop.x + crop.width, crop.y + crop.height ];
+
+		if(forSvg) {
+			pTopLeft = this.arrayDividedBy100(pTopLeft);
+			pTopRight = this.arrayDividedBy100(pTopRight);
+			pBottomLeft = this.arrayDividedBy100(pBottomLeft);
+			pBottomRight = this.arrayDividedBy100(pBottomRight);
+		} else {
+			pTopLeft = this.arrayToPercent(pTopLeft);
+			pTopRight = this.arrayToPercent(pTopRight);
+			pBottomLeft = this.arrayToPercent(pBottomLeft);
+			pBottomRight = this.arrayToPercent(pBottomRight);
 		}
+		return {
+			top: {
+				left: pTopLeft,
+				right: pTopRight
+			},
+			bottom: {
+				left: pBottomLeft,
+				right: pBottomRight
+			}
+		};
+	},
 
-		if (crop.y + bottom === 100) {
-			bottom -= 0.00001;
-		}
+	getImageClipStyle() {
```

Comment:
It's still used. But I wonder if we need it at all. We might get away with url#ReactCropClipPath for both webkit and non prefixed version. What do you think?